package.json

```diff
@@ -58,6 +58,7 @@
     "markdown-it": "8.4.2",
     "markdown-it-anchor": "5.0.2",
     "markdown-it-emoji": "1.4.0",
+    "markdown-it-external-links": "0.0.6",
```

Comment:
You probably want to use git rebase to remove the commit that modifies this file and package-lock.

My normal method would be to do something like `git rebase -i HEAD^^^^^^^` and then removing the line in my editor. 

(sorry if you already know this!)

Reply:
good eye, i forgot that i had imported it